Build fails: missing capabilities field in ProviderInfo

## Description

CI fails on main branch with:

```
error[E0063]: missing field `capabilities` in initializer of `carina_provider_protocol::ProviderInfo`
  --> src/main.rs:52:9
```

The `ProviderInfo` struct in `carina-provider-protocol` (from the main carina repo) was updated to include a `capabilities` field, but `carina-provider-awscc` has not been updated to provide it.

## Steps to reproduce

```bash
cargo check
```

## Expected behavior

Build should succeed.

## Root cause

`carina-provider-protocol::ProviderInfo` added a `capabilities` field that this provider does not yet set.

--- a/src/main.rs
+++ b/src/main.rs
@@ -52,6 +52,7 @@ impl CarinaProvider for AwsccProcessProvider {
         proto::ProviderInfo {
             name: "awscc".into(),
             display_name: "AWS Cloud Control provider".into(),
+            capabilities: vec![],
         }
     }
 

